docs/wiki/Scenario_Low_Power.md

```diff
@@ -0,0 +1,122 @@
+
+### Part B: Finding the Cause
+Now that the issue has been identified and temporarily fixed, the next move should be to determine what may have caused the problem. The first two places that should come to mind to look are a faulty RTS table which has 
+the switch being activated, or an errant ground command. In this case, if we check the script that was run to set up the pass, you may notice an errant switch command, which turns on switch 7. This seems to be the point 
+of failure here, so you should assure it is removed or fixed for future passes.
+
+![lowPower_ScriptError](https://github.com/user-attachments/assets/0733a212-6fd2-42a8-8b92-7312f8b14086)
+
+If it was *not* in the ground script, then you would likely wish to assure that backup versions of the RTS tables that have been checked for the error are sent up to overwrite any currently onboard, as discussed in the 
+In-Flight Patching Scenario. Though, with the error identified, and the speed of discharged observed, the Operations Team deems it wise to add in some sort of failsafe to assure operator error cannot lead to the complete 
+death of the craft.
+
+## Step 3 - Planning the Failsafe
+As emphasized in previous scenarios, before any changes to mission behavior are made, one should consider what exactly needs to be done, how to do it, where the changes need to be made, and any new edge cases that may 
+arise from this new behavior.
+
+### Part A: Determining Scope
+In this case, we want to add a new case where, if the battery gets below a critical level, it will go into a known, safe state. We may also want to make sure any existing power-related triggers go to a fully determined 
+EPS state, though that is up to your discretion. The Avionics Team reports that the battery being used should be recoverable as long as it does not drop below 25%. Thus, 40% is deemed a safe cutoff margin to go into a 
+known safe state in the case of extreme power loss while still being able to survive a night and recharge. 
+
+This means we will need a new LC Watchpoint for 40% power, designed similarly to the existing 60% watchpoint, and we will need to assure it is activated when entering Science Mode. Then, we need to create a new RTS that 
+will transition to Safe Mode if the watchpoint is triggered. Additionally, if it *is* determined that we want to add additional safety measures to our existing low power Science Pause at 60%, we would need to modify that 
+RTS as well.
+
+### Part B: Determining Modifications for Behavior
+Now that we've determined the scope, we need to consider what is necessary for these changes. The LC tables and RTS tables will need to be modified or added as shown in the FDC Check Scenario to add a new watchpoint at 
+40% power, and to assure it is activated. Then, any safe mode transitions need to add new commands added to turn off any switches that are not being currently. Currently, only switches 0 and 1 are being turned off, so 
```

Comment:
Change "need to add new commands added to turn off any switches that are not being currently."
to "need to add new commands to turn off any switches that are not currently being used."